scripts/get_tool_dl_url.py

```diff
@@ -161,8 +162,28 @@ def get_latest_assets(repo: str) -> Iterable[ReleaseAsset]:
 
     body = response.json()
 
-    for asset in body["assets"]:
-        yield ReleaseAsset.from_json(asset)
+    highest_version = semver.VersionInfo(0, 0, 0)
+    latest_release = None
+
+    for release in body:
+        version_string = release["tag_name"]
+        if version_string.startswith("v"):
+            version_string = version_string[1:]
+        if version_string.endswith("+stringlabels"):
```

Comment:
I think we do want to test against rc versions, because this way we can catch upcoming errors faster. This is just supporting script, so I would also not spend more time rewriting each statement over and over again.